modules/http/src/main/java/org/glassfish/grizzly/http/HttpCodecFilter.java

```diff
@@ -815,7 +815,8 @@ protected boolean decodeHttpPacketFromBytes(final FilterChainContext ctx,
                 }
                 if (is100Continue(httpPacket)) {
                     if (parsingState.offset < end) {
```

Comment:
can this condition be removed? End should be the same as inputBuffer.limit().

Reply:
@fsgonz Yes, that makes sense! I will remove this if condition since the parsingState.offset is being checked in the while loop now with a lower value(inputBuffer.limit()).

Reply:
We will keep this if condition for now to update other values accordingly as discussed.